src/js/player.js

```diff
@@ -1640,6 +1640,40 @@ class Player extends Component {
   }
 
   /**
+   * Check whether the player can play a given mimetype
+   *
+   * @param {String} type The mimetype to check
+   * @return {String} 'probably', 'maybe', or '' (empty string)
+   * @method canPlayType
+   */
+  canPlayType(type) {
+    let can;
+
+    // Loop through each playback technology in the options order
+    for (var i=0,j=this.options_.techOrder;i<j.length;i++) {
+      let techName = toTitleCase(j[i]);
+      let tech = Component.getComponent(techName);
+
+      // Check if the current tech is defined before continuing
+      if (!tech) {
+        log.error(`The "${techName}" tech is undefined. Skipped browser support check for that tech.`);
+        continue;
+      }
+
+      // Check if the browser supports this technology
+      if (tech.isSupported()) {
+        can = tech.canPlayType(type);
```

Comment:
should the tech's `canPlayType` check whether it is supported itself?
I guess the downside is that you won't know whether it can't because it isn't supported or because it can't play that type if you ask the tech directly.